src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc

```diff
@@ -1,2 +1,192 @@
 :relative-path: ../../../
-include::{docdir}/variables.adoc[]
\ No newline at end of file
+include::{docdir}/variables.adoc[]
+
+In this part we will explain how to use the **Standard User Workflow** in the nominal case, i.e. using the default implementations of each parts of the workflow (For example, the default implementation to send the confirmation email use the _Mailjet_ API).
```

Comment:
"nominal case" may not be the right words as it suggests that there are error cases that will not be treated. I think "with default behavior" is better because in "default behavior", we handle nominal case (successful signup + sucessful login) and also error cases (signup failure, email failure, login failure...).